BondarenkoBE/lab_5/ConsoleApplication20.cpp

```diff
@@ -70,6 +70,7 @@ int main()
 		for (int j = 0; j < n1; j++)
 			cin >> arr1[i][j];
 	}
+	cin.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
```

Comment:
Why it's necessary?

Reply:
Keyboard buffer cleaning, for that you do not accidentally write anything extra to the array/select an action